examples/vite-rsc/app/root.tsx

```
@@ -57,13 +57,13 @@ function App(props: { url: URL }) {
         <li>
           Edit <code>src/root.tsx</code> to test server HMR.
         </li>
-        {/* <li>
+        <li>
           Visit{" "}
-          <a href="?__rsc" target="_blank">
-            <code>?__rsc</code>
+          <a href="./_.rsc" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)